test/distributed/pipeline/sync/test_pipe.py

```diff
@@ -650,3 +636,19 @@ def forward(self, x):
     # Partition #1:    000! 111! 222!
     #
     assert timeline == [(0, 0), (1, 0), (0, 1), (2, 0), (1, 1), (2, 1)]
+
+@pytest.mark.parametrize("checkpoint", ["never", "always", "except_last"])
+@skip_if_no_cuda
+def test_multiple_inputs(checkpoint, setup_rpc):
+    class Module1(nn.Module):
+        def forward(self, a, b, c):
+            return a + b + c, a * b * c
+
+    class Module2(nn.Module):
+        def forward(self, a, b):
+            return a + b
+
+    model = Pipe(nn.Sequential(Module1().cuda(0), Module2().cuda(0)), chunks=2, checkpoint=checkpoint)
```

Comment:
should change to PipeSequential

Reply:
PipeSequential should only be used internally inside the Pipe API right? I would not expect users to use PipeSequential.